### PR TITLE
Cloudwallet Intergation

### PR DESCRIPTION
**Pull request by mbwhite**
- Implementation of the cloud wallets
- Memory and File System provides included in main repo
- Documentation coming in separate PR

### CODE REVIEW

File: packages/composer-wallet-inmemory/.eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    strict: error
+    no-var: error
+    dot-notation: error
+    no-tabs: error
+    no-trailing-spaces: error
+    no-use-before-define: error
+    no-useless-call: error
+    no-with: error
+    operator-linebreak: error
+#    require-jsdoc:
```
Comment (caroline-church) [inline packages/composer-wallet-inmemory/.eslintrc.yml]: can you uncomment this too please

File: packages/composer-wallet-inmemory/lib/memorywallet.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        let list = a.map( (e) => {
+            return e.replace(this.namePrefix+path.sep,'');
+        } );
+        return Promise.resolve(list);
+    }
+
+    /**
+     * Check to see if the named credentials are in
+     * the wallet.
+     * @abstract
```
Comment (caroline-church) [inline packages/composer-wallet-inmemory/lib/memorywallet.js]: why does this have @abstract and has an implementation

File: packages/composer-wallet-filesystem/.eslintrc.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    strict: error
+    no-var: error
+    dot-notation: error
+    no-tabs: error
+    no-trailing-spaces: error
+    no-use-before-define: error
+    no-useless-call: error
+    no-with: error
+    operator-linebreak: error
+#    require-jsdoc:
```
[the remark this answers is not in the document]
Comment (mbwhite) [reply packages/composer-wallet-filesystem/.eslintrc.yml]: They where copied from one module... but I've replaced the file with a fresh copy

File: packages/composer-wallet-inmemory/.eslintrc.yml (hunk 1 above)
Comment (mbwhite) [reply packages/composer-wallet-inmemory/.eslintrc.yml]: They where copied from one module... but I've replaced the file with a fresh copy

File: packages/composer-wallet-inmemory/lib/memorywallet.js (hunk 2 above)
Comment (mbwhite) [reply packages/composer-wallet-inmemory/lib/memorywallet.js]: it shouldn't.. removed abstract

**Pull request merged**